packages/pds-ember/addon/components/pds/toolbar/stories/1_patterns.stories.js

```diff
@@ -0,0 +1,84 @@
+import hbs from 'htmlbars-inline-precompile'
+import DocsPage, { TITLE } from '../docs.mdx'
+
+export default {
+  title: `${TITLE} / Patterns`,
+  component: 'PdsToolbar',
+  parameters: { docs: { page: DocsPage } },
+}
+
+export const Lists = () => ({
+  template: hbs`
+    <Pds::Toolbar as |T|>
+      <T.Start>
+        <Pds::Input
+          @type="search"
+          id="filter"
+          placeholder="Filter"
+          aria-label="filter content"
+        />
+      </T.Start>
+
+      <T.End>
+        <a href="#" class="pds--cta pds--ghost pds--iconEnd">
```

Comment:
Is it worth pulling in `<Docs::CtaLink>` here?

Reply:
I'm torn, because `<Docs::CtaLink>` was created to DRY up stories for the CTA Link, but it's not a true equivalent to the actual `<Pds::CtaLink>` component.  I wish `@storybook/ember` had a basic Ember app with a couple of dummy routes wired up so that I can avoid using `<Docs::CtaLink>` altogether.

Reply:
Hmm, there may be a way to do this. We register a route class in some of our tests to do this - and I think ember storybook is really similar to the ember testing environment. I can look into it.